hack/machineset.sh

```diff
@@ -257,6 +307,9 @@ case "$provider" in
     azure)
       ms=$(get_azure_ms $infraID $region $az $provider)
     ;;
+    gce)
+      ms=$(get_gcp_ms $infraID $region $az $provider)
```

Comment:
Given the provider is `gce`, does it makes sense to rename it as `get_gce_ms` ?

Reply:
even though the platform value is `gce` all documentation refers to the platform as GCP so i'd prefer to leave it as is